Makefile

```
@@ -31,3 +31,12 @@ build_static:
 
 image:
 	docker build -t mailbadger/app:latest .
+
+run-mailbadger:
```

By convention we should use snake case here.

```suggestion
run_mailbadger:
```